### PR TITLE
mv oc projects tests to separate file

### PR DESCRIPTION
**Pull request by juanvallejo**
Break `oc projects` tests to separate file

### CODE REVIEW

**Comment by juanvallejo** [comment]
[test]

**Comment by juanvallejo** [comment]
@stevekuznetsov or @fabianofranz could you take a look please?

File: test/cmd/projects.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/projects"
+os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
+os::cmd::expect_success_and_text 'oc projects' 'You have access'
```
Comment (stevekuznetsov) [inline test/cmd/projects.sh]: Don't use this test -- see https://github.com/openshift/origin/pull/10047#discussion_r72458000

**Comment by stevekuznetsov** [comment]
LGTM on green tests -- squash down

**Comment by juanvallejo** [comment]
re[test]

**Comment by fabianofranz** [comment]
LGTM [ḿerge]

**Pull request merged**